README

```diff
@@ -216,6 +216,37 @@ existing ``dict``, modifying it using the API and serializing it back to a
     # Convert back to dict to plug back into existing code
     body = s.to_dict()
 
+Development
+-----------
+
+To install all of the dependencies necessary for development, run:
+
+.. code:: bash
+
+    $ pip install -e .[develop]
+
+To run all of the tests for ``elasticsearch-dsl-py``, run:
+
+.. code::bash
+
+    $ python setup.py test
+
+Alternatively, it is possible to use ``pytest`` to run subsets of the test
+suite. Some examples can be seen below:
+
+.. code::bash
+
+    # Run all of the tests in `test_elasticsearch_dsl/test_analysis.py`
+    $ pytest test_elasticsearch_dsl/test_analysis.py
+
+    # Run only the `test_analyzer_serializes_as_name` test.
+    $ pytest test_elasticsearch_dsl/test_analysis.py::test_analyzer_serializes_as_name
+
+If trying to run a test from `test_elasticsearch_dsl/test_integration/`,
+ensure that you have a test instance of ElasticSearch running for which a
+test connection can be created. Integration tests will be skipped if it is
+not possible to create an ElasticSearch connection.
```

Comment:
It would be good to mention that the luster should be empty and destructive changes will occur, a big warning is probably in place.

Also pointing to the `TEST_ES_SERVER` that can be used to point to different hostname/port might be useful - https://github.com/elastic/elasticsearch-py/blob/master/elasticsearch/helpers/test.py#L12-L31